src/test/java/com/boot/amazon/service/impl/FileServiceImplTest.java

```diff
@@ -0,0 +1,40 @@
+package com.boot.amazon.service.impl;
+
+import com.boot.amazon.service.FileService;
+import java.io.File;
+import java.io.IOException;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class FileServiceImplTest {
+    private static final String LOAD_FROM_URL =
+            "https://spring-boot-aws-revievers.s3.eu-central-1.amazonaws.com/Reviews.csv";
+    private static final String SAVE_TO_PATH = "src/file.storage/file.txt";
```

Comment:
let's better save it to resources package. It's ease to get file from there only by name without hardcoding the whole path

Reply:
Cool! Moved it there